content/news/2025-01-22_rocur-relaunch/index.en.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "We're Launching the R-Ladies Rotating Curation (RoCur) Account on Bluesky!"
+author: "RoCur team"
+type: blog
+date: "2025-01-22"
+image: img/rladies-bluesky.jpg
+tags: 
+- rocur
+categories: 
+- R-Ladies
+summary: We're looking for curators for the We Are R-Ladies Bluesky account!
+slug: rocur-relaunch-2025
+---
+
+![Screenshot of We Are R-Ladies on Bluesky](img/rladies-bluesky.jpg)
+
+We are absolutely thrilled to announce the R-Ladies Rotating Curation (RoCur) Bluesky account [bsky.app/profile/weare.rladies.org](https://bsky.app/profile/weare.rladies.org) is now live! Each week, a new member of the R-Ladies community will take over the account to talk about all things R. This will be a fantastic opportunity for individuals who identify as a minority gender in the R community to amplify their voices and connect with others in the greater data science community.
```

Comment:
For future PRs I'd recommend starting a new line after each sentence. 
You can use https://github.com/ropenscilabs/aeolus to fix documents :smile_cat: